src/test/java/com/amazon/ion/impl/IonReaderContinuableTopLevelBinaryTest.java

```diff
@@ -5934,5 +5934,54 @@ public void prefixedAnnotatedContainerInsideDelimitedAnnotatedContainerPreserves
         closeAndCount();
     }
 
+
+    @ParameterizedTest(name = "[{index}] {0}")
+    @CsvSource({
+        // (:values 0) (:values 1)
+        "EF 01 01 60 EF 01 01 61 01",
+        // (:values (: values 0)) 1
+        "EF 01 01 EF 01 01 60 61 01",
+        // (:values) 0 (:values) 1
+        "EF 01 00 60 EF 01 00 61 01",
+        // TODO: Determine why the state isn't being properly set after invoking a macro that produces nothing
+        // (:values) (:values 0) 1
+        // "EF 01 00 EF 01 01 60 61 01",
+    })
+    public void invokeValuesUsingSystemMacroOpcode(String bytes) throws Exception {
+        invokeValuesUsingSystemMacroOpcodeHelper(true, bytes);
+        invokeValuesUsingSystemMacroOpcodeHelper(false, bytes);
+    }
+
+    @ParameterizedTest(name = "[{index}] {0}")
+    @CsvSource({
+        // (:values (:: ) ) 0 1
+        "EF 01 02 01 F0 60 61 01",
+        // (:values 0 1) // using length-prefixed expression group
```

Comment:
```suggestion
        // (:values (:: 0 1)) // using length-prefixed expression group
```
Is this right?

Reply:
Either one is legal since we can use the implicit rest param for the argument to `values`. I've only used `(:: )` in some places to make it clear that the encoding is an empty expression group or a single-arg expression group.